analysis/webservice/webapp.py

```diff
@@ -25,10 +25,16 @@
 import pkg_resources
 import tornado.web
 from tornado.options import define, options, parse_command_line
-
 from webservice import NexusHandler
 from webservice.webmodel import NexusRequestObject, NexusProcessingException
 
+logging.basicConfig(
```

Comment:
Care should be taken when configuring `logging` in globally like this. Now, whenever this module is imported, logging will be configured (unless it was already configured by some other module which also does this). Generally, it is better to configure logging in `__main__`, is there a reason you wanted to move it?

Reply:
Ok thanks Franck, I moved back this section in __main__
The reason why I moved it initially is that I wanted to use the log outside the __main__. I created a local one instead.